### ISSUE TITLE
tf-stanford-tutorials/examples/03_logistic_regression_mnist_sol.py

### ISSUE THREAD

**Issue by lucky6qi** [chiphuyen/tf-stanford-tutorials#31]
Line 76 should be  

`loss_batch, logits_batch = sess.run([loss, logits], feed_dict={X: X_batch, Y:Y_batch}) `

We need to test it based on test data, not train it based on test data.

**Comment by chiphuyen** [comment]
You're right. My bad. Fixed.
